src/Common/gui/NinjamPanel.cpp

```diff
@@ -256,9 +258,34 @@ int NinjamPanel::getPanSliderMaximumValue() const
     return ui->panSlider->maximum();
 }
 
-int NinjamPanel::getCurrentBeatsPerAccent() const
+int NinjamPanel::getAccentBeatsComboValue() const
 {
-    return ui->comboBeatsPerAccent->currentData().toInt();
+    return ui->comboAccentBeats->currentData().toInt();
+}
+
+bool NinjamPanel::isAccentBeatsEnabled() const
+{
+    return ui->lineEditAccentBeats->isEnabled();
+}
+
+void NinjamPanel::setAccentBeatsEnabled(bool value)
+{
+    qCDebug(jtNinjamGUI) << "NinjamPanel::setAccentBeatsEnabled " << value;
+    ui->lineEditAccentBeats->setEnabled(value);
+}
+
+QString NinjamPanel::getAccentBeatsText() const
+{
+    return ui->lineEditAccentBeats->text();
+}
+
+void NinjamPanel::setAccentBeatsText(QString value)
+{
+    qCDebug(jtNinjamGUI) << "NinjamPanel::setCurrentAccentBeats " << value;
+    ui->lineEditAccentBeats->blockSignals(false);
```

Comment:
@pljones , this is this problem. The block/unblock signals commands are inverted. Block first, and unblock in the end. Tested here and works fine after fix the block/unblock.

Reply:
You're right - but that didn't fix the space bar issue (still goes to the host).